kobo/client/commands/cmd_create_worker.py

```diff
@@ -0,0 +1,34 @@
+import sys
+from xmlrpc.client import Fault
+import pprint
+
+from kobo.client import ClientCommand
+
+
+class Create_Worker(ClientCommand):
+    """create a worker"""
+    enabled = True
+    admin = True
+
+
+    def options(self):
+        self.parser.usage = "%%prog %s [--create] worker_name [worker_name]" % self.normalized_name
+
+    def run(self, *args, **kwargs):
+        create = kwargs.pop("create", False)
+
+        if not args:
+            self.parser.error("No worker name specified.")
+
+        username = kwargs.pop("username", None)
+        password = kwargs.pop("password", None)
+        hub = kwargs.pop("hub", None)
+        workers = args
+
+        self.set_hub(username, password, hub)
+        for worker in workers:
+            try:
+                new_worker = self.hub.client.create_worker(worker)
+                pprint.pprint(new_worker)
+            except Fault as ex:
+                print(repr(ex), file=sys.stderr)
```

Comment:
With this implementation it's still going to exit with a 0 exit code even if worker creation fails.

Is that intentional?  It seems to make any kind of scripted usage more difficult, as if you do `my-client create-worker foo`, you can't then proceed under the assumption that `foo` exists even if the command succeeded.

Reply:
I have modified the code to exit on first xmlrpc failure.